connector/src/main/java/org/geysermc/connector/entity/player/SessionPlayerEntity.java

```diff
@@ -74,9 +74,9 @@ public void spawnEntity(GeyserSession session) {
     }
 
     @Override
-    public void moveAbsolute(GeyserSession session, Vector3f position, Vector3f rotation, boolean isOnGround, boolean teleported) {
-        session.getCollisionManager().updatePlayerBoundingBox(position);
-        super.moveAbsolute(session, position, rotation, isOnGround, teleported);
+    public void moveRelative(GeyserSession session, double relX, double relY, double relZ, Vector3f rotation, boolean isOnGround) {
+        super.moveRelative(session, relX, relY, relZ, rotation, isOnGround);
+        session.getCollisionManager().updatePlayerBoundingBox(this.position.down(entityType.getOffset()));
```

Comment:
For the future: I think it could be micro-efficient to have a updatePlayerBoundingBox method that takes in three doubles.